examples/pdf/AnalyticED_example.cpp

```diff
@@ -1,33 +1,33 @@
 /*
   Demonstrates the use of an Analytic pdf, using a 2D gaussian function
 */
-#include <AnalyticPdf.h>
+#include <AnalyticED.h>
 #include <Gaussian.h>
 int main(){
     /* We can use any function that implements the IntegrableFunction 
        interface, here just use a gaussian
      */
     Gaussian gaus(std::vector<double>(2, 0), std::vector<double>(2, 1));
     
-    AnalyticPdf gausPdf(&gaus); // takes ownership of a copy of gaussian
+    AnalyticED gausPdf("gausPDF", &gaus); // takes ownership of a copy of gaussian
 
     // The dimensionality of the pdf now matches the underlying function
     std::cout << "Pdf is " << gausPdf.GetNDims() << std::endl;
 
     // now we can call it directly
-    std::cout << " gausPdf(0, 0) = "
-              << gausPdf(std::vector<double>(2, 0)) << std::endl;
+    std::cout << " gausPdf(2, 0) = "
```

Comment:
Yeah, I am not sure what was meant by calling it directly here, so probably a good shout with probability. Could you put gausPdf.Probabilty(2,0) on this line in the quotes too, so that it's printed?

Reply:
That seems sensible.